test/integration-test.sh

```diff
@@ -60,4 +60,4 @@ case $1 in
 esac
 
 cd "$DIR"
-docker run  -e TAPE_LOGLEVEL=debug --network host -v $PWD:/config tape tape $CONFIG_FILE 500
+docker run  -e TAPE_LOGLEVEL=debug --network host -v $PWD:/config tape tape -c $CONFIG_FILE -n 500
```

Comment:
Should we consider cover with default values feature in our e2e testing or CI testing?
https://github.com/guoger/tape/pull/119/files#r537962402
as we adding default values, I hope it been tested.